src/sentry/integrations/github/issues.py

```diff
@@ -173,7 +171,7 @@ def get_create_issue_config(
         ]
 
     def create_issue(self, data: Mapping[str, Any], **kwargs: Any) -> Mapping[str, Any]:
-        with self.record_event(SourceCodeIssueIntegrationInteractionType.CREATE_ISSUE).capture():
+        with self.record_event(SCMIntegrationInteractionType.CREATE_ISSUE).capture():
```

Comment:
This is an issue basic feature right? My current PR adds an enum value for other ticketing integration behavior, so it might be better to use this one instead: https://github.com/getsentry/sentry/pull/80719/files (see integrations/project_management/metrics.py).

This will allow us to create blanket metrics and alerts for all ticket creation a bit easier, since this seems like a high priority feature to track.

Also, this is probably related to our conversation yesterday: Won't this be covered already by the calling code? I'm adding metric gathering to `sentry/rules/actions/integrations/create_ticket/utils.py`, which is the only calling code I see for this, so we may be able to remove this entirely.

Reply:
i think this should be covered by calling code, you're right. i wasn't aware if the PR was up yet, but i can remove the extra stuff here!